src/platform/AT32/mk/AT32F4.mk

```diff
@@ -63,6 +63,7 @@ DEVICE_STDPERIPH_SRC = $(STDPERIPH_SRC)
 INCLUDE_DIRS    := $(INCLUDE_DIRS) \
                    $(TARGET_PLATFORM_DIR)/startup \
                    $(TARGET_PLATFORM_DIR) \
+                   $(PLATFORM_DIR)/common/stm32 \
```

Comment:
Why use stm32 here as it is shared with APM, AT32

Reply:
Because AT and APM are derivatives (clones) of STM.